gfsphysics/GFS_layer/GFS_typedefs.F90

```diff
@@ -744,6 +741,12 @@ real (kind=kind_phys), pointer :: prsik (:,:) => null()   !< Exner function at i
     logical              :: trans_trac      !< flag for convective transport of tracers (RAS, CS, or SAMF)
     logical              :: old_monin       !< flag for diff monin schemes
     logical              :: cnvgwd          !< flag for conv gravity wave drag
+#ifdef CCPP
+    integer              :: gwd_opt         !< gwd_opt = 1  => original GFS gwd (gwdps.f)
```

Comment:
Where does the UGW in GFSv16 fall under this category?

Reply:
The UGW and original GFS gravity wave drag are controlled as before by their own variables (cnvgwd, do_tofd, ...) - this gwd_opt is only effective when the GSD drag suite is in the suite definition file. Once the GFS-UGW gravity wave drag code has converged to a clean solution (the current way of calling gwdps from inside UGWP is a little confusing), we should straighten out all the options and control switches in my opinion.